blockchain/chain/src/store/errors.rs

```diff
@@ -65,8 +64,8 @@ impl From<String> for Error {
     }
 }
 
-impl From<Box<dyn StdError>> for Error {
-    fn from(e: Box<dyn StdError>) -> Self {
+impl From<anyhow::Error> for Error {
+    fn from(e: anyhow::Error) -> Self {
         Error::Other(e.to_string())
     }
 }
```

Comment:
Maybe `Error::Other(String)` should be `Error::Other(anyhow::Error)`

Reply:
Hmm this has a big impact on all code using `Err(Error::Other("...`. ie [here](https://github.com/ChainSafe/forest/blob/adfe507967c54aa28a937ae6c74a79379e30fac2/blockchain/chain/src/store/chain_store.rs#L156).